File: benchmark/workloads/go/tableLoop.go
package main

import "fmt"

func main() {
	values := make([]uint64, 1_000_000)
	for index := range values {
		values[index] = uint64(index + 1)
	}
	var total uint64
	for _, value := range values {
		total += value
	}
	fmt.Println(total)
}

File: benchmark/workloads/rust/tableLoop.rs
fn main() {
    let mut values = vec![0_u64; 1_000_000];
    for (index, value) in values.iter_mut().enumerate() {
        *value = index as u64 + 1;
    }
    let total: u64 = values.iter().sum();
    println!("{total}");
}
